src/Config/Config.ts

```diff
@@ -36,6 +37,13 @@ export default class Config implements IConfigParams {
       throw new Error('Please set the providerUrl in the config object.');
     }
 
+    this.economicStrategy = params.economicStrategy || {
```

Comment:
will the values be visible to the user from here?
We need the user to see the default values if they open the `Economic strategy` section when trying to activate claiming

Reply:
This is up to the clients (cli/dapp) to decide how they want to show it to the user. They can always access `config.economicStrategy.maxDeposit` and check, etc.

Reply:
We could also expose it so they can show it to the user even before setting the Config object.

Reply:
I would like to avoid that though, adds too much complexity.